### DISCUSSION TITLE
Clarity on changelog of PHP 7.3.33-10+0~20230409.104+debian10~1.gbp88ff76 on Debian Buster

### DISCUSSION THREAD

**Discussion by mig5** [oerdnj/deb.sury.org#1944] [Q&A] [answered]
On Debian Buster, just ran some routine patching and saw this upgrade of PHP 7.3:

from:
`7.3.33-10+0~20230214.103+debian10~1.gbp88ff76`

to:
`7.3.33-10+0~20230409.104+debian10~1.gbp88ff76`

I looked at `/usr/share/doc/php7.3-common/changelog.Debian.gz` afterward, to see what the change was (as I didn't see any new release upstream):

```
php7.3 (7.3.33-10+0~20230409.104+debian10~1.gbp88ff76) unstable; urgency=medium

  ** SNAPSHOT build @88ff76bda1af9b7a674d7221b0a14cc6c9997390 **

  * UNRELEASED

 -- Ondřej Surý <ondrej@sury.org>  Sun, 09 Apr 2023 18:50:14 +0200

```

If it is unreleased, how come it arrived in the debian repository? Can we get clarification as to what the changes actually were, if any?

Thanks!

**Comment by todeveni** [comment, marked as answer]
I guess that changelog might be autogenerated. And this version update is due to rebuilding of packages. My guess: bookworm support 🥳

That "snapshot" commit [88ff76bda1af9b7a674d7221b0a14cc6c9997390](https://salsa.debian.org/php-team/php/-/commit/88ff76bda1af9b7a674d7221b0a14cc6c9997390) is the latest 7.3.33-10 version from february. So there's no actual changes in the PHP version itself.

**Reply by oerdnj** [reply]
That’s how debian-jenkins-glue generates changelogs.

Also it can’t build only missing packages, it needs to always rebuild the whole set.

**Reply by mig5** [reply]
Thanks @todeveni!
